### ISSUE TITLE
new official group on mangadex

### ISSUE THREAD

**Issue by fadderman** [tachiyomiorg/tachiyomi-extensions#14054]
### Source information and language

mangadex 1.3.173

### Steps to reproduce

Updated to 0.14.1
Updated mangadex to 1.3.173

### Expected behavior

Groups blocked by default should be blocked 

### Actual behavior

Doesn't block group by uuid 

### Tachiyomi version

0.14.1

### Android version

android 11

### Other details

Found the problem new official group was spamming mangadex, need to add caa63201-4a17-4b7f-95ff-ed884a2b7e60 to the default filter

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.14.1](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by fadderman** [comment]
Retitled the problem after i found the solution

**Comment by curche** [comment]
do you have a link to any particular manga you were having issues with?
or does anything in this link work https://mangadex.org/group/caa63201-4a17-4b7f-95ff-ed884a2b7e60?tab=titles

**Comment by fadderman** [comment]
Sorry for the bad ticketet, I wrote it then came to checkboxes before submit and i went back and did a thourogh check of what I thought was the bug was. 
Found out that what I thought was billibilli passing through the block group filter was a new actor in the market that was marketing like billibilli. 
So I went back and filled in other details with the correct solution to my problem and forgot to change the rest of the boxes. 
Had also forgot to put a title on the ticket so i just wrote it just before submitting the ticket. 
So you now have a ticket spilt in 2 about what the problem is, one half of the ticket seems to think there is a problem with group block filter, and the other half of the ticket about that INKR Comics is like MangaPlus, BilliBilli and the others in the default and should be added to the plugins default filter
